provisioning/security/tpm-provider-emulator/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/hsm/SecurityProviderTPMEmulator.java

```diff
@@ -511,10 +511,9 @@ else if (rc != TPM_RC.HANDLE)
     /**
      * Getter for extracting EndorsementKey from TPM
      * @return The Endorsement Key from TPM
-     * @throws SecurityProviderException If endorsement key could not be extracted.
      */
     @Override
-    public byte[] getEndorsementKey() throws SecurityProviderException
```

Comment:
This class is arguably only for testing purposes, and customers aren't expected to use it outside of their own tests, but we may want to not break it just to be safe